lib/classifier.js

```diff
@@ -313,7 +313,7 @@ async function findByObjectDetector (driver, logger, label, allowWeakerMatches)
 
   // TODO don't hardcode screenshot path; instead we want to find a way to pass
   // an image data buffer to the native c code
-  const screenshotPath = "/Users/jlipps/Desktop/test.png";
+  const screenshotPath = `${__dirname}/${new Date().toUTCString()}_classifiertest.png`;
```

Comment:
can we use `path.resolve(__dirname, `${new Date().toUTCString()}_classifiertest.png`)` to ensure it works cross platform?

Reply:
Ah sure I forget that. Will do it